client/src/api/block_builder/input_selection/sender_issuer.rs

```diff
@@ -230,12 +230,12 @@ fn get_required_addresses_for_sender_and_issuer(
     // Addresses in the inputs that will be unlocked in the transaction
     let mut unlocked_addresses = HashSet::new();
     for input_signing_data in selected_inputs {
-        let alias_state_transition = is_alias_state_transition(input_signing_data, outputs)?;
+        let alias_transition = is_alias_transition(input_signing_data, outputs);
         let (required_unlock_address, unlocked_alias_or_nft_address) =
             input_signing_data.output.required_and_unlocked_address(
                 current_time,
                 input_signing_data.output_id(),
-                alias_state_transition.unwrap_or((false, false)).0,
+                Some(alias_transition.unwrap_or((AliasTransition::Governance, false)).0),
```

Comment:
This was previously false so I put Governance, but why are we defaulting to governance again ?

Reply:
Defaulting to governance was because if `let alias_state_transition = is_alias_state_transition(input_signing_data, outputs)?;` returns None, then it meant we would burn it
Might not really make sense anymore now, since the burning is provided different

Reply:
I removed it, and just returned the result from is_alias_transition